skynet/bandwidth.go

```diff
@@ -52,10 +66,23 @@ func BandwidthDownloadCost(size int64) int64 {
 	return PriceBandwidthDownloadBase + chunks*PriceBandwidthDownloadIncrement
 }
 
+// RawStorageUsed calculates how much storage an upload with a given size
+// actually uses. This method returns the total underlying storage used and not
+// the adjusted number users see. Users see adjusted numbers in order to
+// shield them from the complexity of base/chunk redundancy.
+func RawStorageUsed(uploadSize int64) int64 {
+	baseSectorStorage := int64(PriceStorageUploadBase * RedundancyBaseSector)
+	chunkStorage := numChunks(uploadSize) * PriceStorageUploadIncrement * RedundancyChunk
+	return baseSectorStorage + chunkStorage
+}
+
 // StorageUsed calculates how much storage an upload with a given size actually
-// uses.
+// uses. This method returns user-facing values. For the raw storage value
+// use RawStorageUsed().
 func StorageUsed(uploadSize int64) int64 {
-	return PriceStorageUploadBase + numChunks(uploadSize)*PriceStorageUploadIncrement
+	baseSectorStorage := int64(PriceStorageUploadBase * RedundancyBaseSector)
+	chunkStorage := numChunks(uploadSize) * PriceStorageUploadIncrement * RedundancyChunk
+	return (baseSectorStorage + chunkStorage) / RedundancyAPIDivisor
```

Comment:
Can this whole function just be `RawStorageUsed(uploadSize) / RedundancyAPIDivisor`?